src/pages/todo/components/TodoListItem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router'
+
+class TodoListItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {hidden: true}
+  }
+  
+  componentDidUpdate() {
+    this.refs.textbox.focus()
+  }
+  _showInput() {
+    this.setState({hidden: false})
+  }
+
+  _hideInput() {
+    this.setState({hidden: true})
+  }
+
+  render() {
+    const { todo } = this.props;
+    const { hidden } = this.state;
+    const style = {
+      label: {display: !hidden ? 'none' : 'block'},
+      textbox: {display: hidden ? 'none' : 'block'},
+    }
+    return (
+      <div>
```

Comment:
Dear Mr @kenji4569 :
Sorry, in this case, i intend use for update, so it not complete yet.